src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/TableSchemaObject.java

```diff
@@ -115,4 +127,127 @@ public static TableSchemaObject from(TableMetadata tableMetadata, ObjectMapper o
     }
     return new TableSchemaObject(tableMetadata, vectorConfig);
   }
+
+  /**
+   * Convert table schema object to table response which is returned as response for `listTables`
+   *
+   * @return
+   */
+  public TableResponse toTableResponse() {
+    String tableName = removeQuotes(tableMetadata().getName());
```

Comment:
As per my later note, I think we should add a helper method in "CqlIdentifierUtil", and that should simply be something like:

```
public static String externalRepresentation(CqlIdentifier id) {
  return id.asInternal();
}
```

(assuming that works -- but if not, adding handling)